resolwe_bio/docker_images/dnaseq/README.md

```diff
@@ -8,6 +8,7 @@ https://hub.docker.com/r/resolwebio/base/) image.
 Included bioinformatics tools:
 ------------------------------
 * CheMut
-* GATK (4.0.11.0)
+* GATK (4.1.2.0)
 * LoFreq (2.1.3.1)
 * Primerclip (v180308)
+* bamclipper
```

Comment:
Add version information (1.1.1) and move to the top (we use alphabetical ordering).

Reply:
Nice catch. Added.